mobile/UI_REFACTORING_ROADMAP.md

```
@@ -0,0 +1,194 @@
+  - `onCancel: () => void`
+  - `onConfirm: () => void`
+  - `confirmLabel?: string`
+  - `loading?: boolean`
+  - `disabled?: boolean`
+
+---
+
+## Tracking
+
+| Phase | Item | Status | Impact | Files |
+|-------|------|--------|--------|-------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)